config/routes.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  devise_for :users
+
+  devise_scope :user do
+    authenticated :user do
+      root to: 'categories#index', as: :authenticated_root
+    end
+
+    unauthenticated do
+      root 'home#index'
+    end
+  end
+
+  resources :categories do
+    resources :atransactions
+  end
```

Comment:
### config/routes.rb

- [OPtiONAL] It would be nice to use only, except, ... options to add only necessary routes in case maybe there are some routes that are not used. 👍🏻